### PR TITLE
ran create-react-app

### PR DESCRIPTION
**Pull request by JaredHerbert11**
Set up front end environment for app

### CODE REVIEW

**Comment by JaredHerbert11** [comment]
This PR was a demonstration

File: laundr_swe/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
```
Comment (chburrows) [inline laundr_swe/.gitignore:11]: Demonstration code comment for pull

**Review by chburrows — CHANGES_REQUESTED** [review]
test review

File: laundr_swe/.gitignore (hunk 1 above)
Comment (JaredHerbert11) [reply laundr_swe/.gitignore:11]: Changes were made

**Pull request merged**